src/plugins/kibana/public/settings/styles/main.less

```diff
@@ -205,26 +205,34 @@ kbn-settings-indices {
   .time-and-pattern > div {}
 }
 
+.wizard-container {
+  padding: 0 10px;
+}
+
 .wizard-step-headings{
-  margin-top: 1em;
+  padding-top: 1em;
```

Comment:
It looks like our code base uses the `px` unit for things like padding and margin, with a few exceptions. I spoke with @panda01 and he suggested we continue to use px for these styles, and only use em for font sizes.